<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="BCTCW-HP-63" name="com.cuetrans.testcases.VehicleMaster_Create" tests="2" failures="0" timestamp="08 Oct 2018 09:09:37 GMT" time="64.433" errors="2">
  <testcase name="afterTest" time="7.994" classname="com.cuetrans.testcases.VehicleMaster_Create">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at org.apache.pdfbox.cos.COSString.<init>(COSString.java:126)
at org.apache.pdfbox.pdmodel.edit.PDPageContentStream.drawString(PDPageContentStream.java:556)
at com.cuetrans.utils.PDFReporter.headerTable(PDFReporter.java:482)
at com.cuetrans.utils.PDFReporter.addTestExecutionOverview(PDFReporter.java:381)
at com.cuetrans.utils.PDFReporter.createDocumentWithMultiplePage(PDFReporter.java:177)
at com.cuetrans.utils.PDFReporter.createPDFReport(PDFReporter.java:529)
at com.cuetrans.wrappers.ProjectWrappers.afterTest(ProjectWrappers.java:230)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- afterTest -->
  <testcase name="loginPass" time="56.439" classname="com.cuetrans.testcases.VehicleMaster_Create">
    <error type="java.lang.RuntimeException" message="Failed">
      <![CDATA[java.lang.RuntimeException: Failed
at com.cuetrans.utils.ReporterExtent.reportStep(ReporterExtent.java:58)
at com.cuetrans.utils.ReporterExtent.reportStep(ReporterExtent.java:23)
at com.cuetrans.wrappers.GenericWrappers.enter(GenericWrappers.java:1116)
at com.cuetrans.pages.CreateVehiclePage.enterVehicleCode(CreateVehiclePage.java:28)
at com.cuetrans.testcases.VehicleMaster_Create.loginPass(VehicleMaster_Create.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- loginPass -->
</testsuite> <!-- com.cuetrans.testcases.VehicleMaster_Create -->
